pkg/pipelines/tekton/pipelines_pac_provider.go

```diff
@@ -133,7 +133,7 @@ func (pp *PipelinesProvider) createLocalPACResources(ctx context.Context, f fn.F
 // also creates PVC for the function source code
 func (pp *PipelinesProvider) createClusterPACResources(ctx context.Context, f fn.Function, metadata pipelines.PacMetadata) error {
 	// figure out pac installation namespace
-	installed, _, err := pac.DetectPACInstallation(ctx, "")
+	installed, _, err := pac.DetectPACInstallation(ctx, "openshift-pipelines")
```

Comment:
rationale: this gives the function hint where to search the PaC info config map so all namespace scan is not required (unprivileged user cannot scan all the namespaces)